packages/koa-metrics/src/timing.ts

```diff
@@ -1,10 +1,10 @@
 import {Context} from 'koa';
 
 export function getQueuingTime(ctx: Context): number | null {
-  const requestQueuingTime = ctx.request.get('X-Request-Start');
-  if (requestQueuingTime) {
+  const requestStartHeader = ctx.request.get('X-Request-Start');
+  if (requestStartHeader) {
     try {
-      return parseInt(requestQueuingTime, 10);
+      return parseInt(requestStartHeader.replace('t=', ''), 10);
```

Comment:
since replace will take care of the case when t= is not there, this should be safe. But I would make it a minor change instead of patch.

Reply:
According to README you sent, I have to page ATC from web foundation to make the release, right>?

Reply:
You mean no need to release a new version?

Reply:
You do need a new version, in the description of the issue thou, you checked off a patch version which is understandable that this is a bug to you. To this library and its consumer thou, the change you made is actually a new feature which would be a minor version bump. Do you agree?

Reply:
gotcha! Agree 👍